src/index.ts

```diff
@@ -13,31 +9,40 @@ const standardOptions = {
   // heartbeatFrequencyMS: 5000,
 }
 
-const Global = {
-  isConnected: null,
+type TGlobal = {
+  isConnected?: boolean
+  checkConnectionOnceTimerID: null
+}
+
+const Global: TGlobal = {
+  isConnected: undefined,
   checkConnectionOnceTimerID: null,
 }
 
-function _getMongoOptionsWithoutDbUri(options) {
-  const dbOptions = { ...standardOptions, ...options }
-  delete dbOptions.dbUri
-  delete dbOptions.logger
-  return dbOptions
+type ConnectOptions = {
```

Comment:
I am unsure of what `ConnectOptions ` should look like, maybe add properties here when we try this package out in a project?